zsl/application/service_application.py

```diff
@@ -56,6 +135,19 @@ def get_version(self):
         else:
             return AtteqServiceFlask.VERSION + ":" + v
 
-if not is_initialized():
-    raise Exception("Can not instantiate ServiceApplication object, the service is not initialized.")
-service_application = AtteqServiceFlask("zsl.application")
+    run_web = Flask.run
+    """Alias for Flask.run"""
+
+    def run_worker(self, *args, **kwargs):
```

Comment:
This needs a deeper discussion about the general architecture and better future proofing. I put this here to allow client to create workers, like you do when you are creating flask application. You don't call some flask tool to run you configuration, you write the app directly and call appropriate methods.